### PR TITLE
Remove ILI9341 SPI buffer range limit

### PR DESCRIPTION
**Pull request by HumJ0218**
fix issue from #1792

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/1802)

### CODE REVIEW

**Comment by krwq** [comment]
please test on at least RPi3 and RPI4

**Comment by pgrawehr** [comment]
> please test on at least RPi3 and RPI4

I don't think that is necessary. To remove this test, we only have to prove that there is a setup where the test is _wrong_, not that there s a setup where it might be necessary. If the latter where true (e.g the maximum SPI buffer size on the RPI3 is 65kb) that test shold not be here.

File: src/devices/Ili9341/Ili9341.cs [hunk 1]
```diff
@@ -53,11 +53,6 @@ public partial class Ili9341 : IDisposable
         /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
         public Ili9341(SpiDevice spiDevice, int dataCommandPin, int resetPin, int backlightPin = -1, int spiBufferSize = DefaultSPIBufferSize, GpioController? gpioController = null, bool shouldDispose = true)
         {
-            if (!InRange((uint)spiBufferSize, 0x1000, 0x10000))
```
Comment (pgrawehr) [inline src/devices/Ili9341/Ili9341.cs:56]: Maybe we should leave a test to make sure the value is positive?

**Review by krwq — APPROVED** [review]
LGTM after checking for buffer size larger than 0

**Review by pgrawehr — APPROVED** [review]

**Pull request merged**